test/common/test_utility.cc

```diff
@@ -0,0 +1,13 @@
+#include <string>
+
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace TestUtil {
+
+envoy_data make_envoy_data(const std::string& s) {
```

Comment:
Can we move this to the data utility file?

Reply:
And name it in line with other mapping utilities, please. :)

Reply:
ya